src/Maestro/Maestro.Web/Api/v2019_01_16/Controllers/SubscriptionsController.cs

```diff
@@ -0,0 +1,330 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Maestro.Contracts;
+using Maestro.Data;
+using Maestro.Data.Models;
+using Microsoft.AspNetCore.ApiPagination;
+using Microsoft.AspNetCore.ApiVersioning;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.ServiceFabric.Actors;
+using Swashbuckle.AspNetCore.Annotations;
+using Channel = Maestro.Data.Models.Channel;
+using Subscription = Maestro.Web.Api.v2019_01_16.Models.Subscription;
+using SubscriptionUpdate = Maestro.Web.Api.v2018_07_16.Models.SubscriptionUpdate;
+using SubscriptionData = Maestro.Web.Api.v2018_07_16.Models.SubscriptionData;
+
+namespace Maestro.Web.Api.v2019_01_16.Controllers
+{
+    public class SubscriptionsController_ApiRemoved : Maestro.Web.Api.v2018_07_16.Controllers.SubscriptionsController
+    {
+        public SubscriptionsController_ApiRemoved(
+            BuildAssetRegistryContext context,
+            BackgroundQueue queue,
+            IDependencyUpdater dependencyUpdater,
+            Func<ActorId, ISubscriptionActor> subscriptionActorFactory)
+            : base(context, queue, dependencyUpdater, subscriptionActorFactory)
+        {
+        }
+
+        [ApiRemoved]
```

Comment:
Same thing here, if the signatures match just override without the [ApiRemoved] thing.